packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+
+#### `scale` property
+
+- `fixed`: Size of the card is fixed.
+- `auto-width`: `width` is set to `fit-content`.
+- `auto-height`: `height` is set to `fit-content`.
+- `auto`: `width` and `height` are set to `fit-content`.
+- `fluid-width`: `width` is set to `100%`.
+- `fluid-height`: `height` is set to `100%`.
+- `fluid`: `width` and `height` are set to `100%`.
+
+### CardHeader
+
+#### Anatomy
+
+![visual anatomy of the CardHeader component](./assets/CardHeader.png)
+
+#### API
+
+| Property    | Values | Default   | Purpose                                           |
+| ----------- | ------ | --------- | ------------------------------------------------- |
+| image       | _slot_ | undefined | Logo or image to better contextualize the content |
+| title       | _slot_ | undefined | Main title of the card                            |
+| description | _slot_ | undefined | Further description of the content                |
+
+### CardBody
+
+This component has no custom props and only ensures the proper positioning of content and itself within the card.
+
+### CardPreview
+
+![visual anatomy of the CardPreview component](./assets/CardPreview.png)
+
+#### API
+
+| Property | Values | Default   | Purpose                                                  |
+| -------- | ------ | --------- | -------------------------------------------------------- |
+| src      | string | undefined | Source of the media to be displayed on the `img` element |
+| alt      | string | undefined | Alternative text to describe the media presented         |
+| logo     | _slot_ | undefined | A logo/file type to add over the image preview           |
+
+### CardFooter
+
+This component has no custom props and only ensures the proper positioning of content and itself within the card.
+
+## Sample Code
+
+### Compact
+
+```jsx
+<Card orientation="horizontal">
+  <CardHeader image={<img src="powerpoint.png" />} title="Team offsite 2020" description="Onedrive > Files" />
+</Card>
+```
+
+### App
+
+```jsx
+<Card>
+  <CardHeader image={<img src="app_logo.png" />} title="App Name" description="Developer" />
+  <CardBody>
+    Donut chocolate bar oat cake. Dragée tiramisu lollipop bear claw. Marshmallow pastry jujubes toffee sugar plum.
+  </CardBody>
+</Card>
+```
+
+### Task
+
+```jsx
+<Card>
+  <CardHeader title="Alert in Teams when a new document is uploaded in channel" description="By Microsoft" />
+  <CardFooter>
+    <span>Automated</span>
+    <span>3290 uses</span>
+  </CardFooter>
+</Card>
+```
+
+### Image
+
+```jsx
+<Card>
+  <CardPreview src="monkey.png" alt="image of monkey" />
+  <CardBody>
+    <div>
+      <span>Length</span>
+      <span>4.6-39.6 IN</span>
+      <span>11.7-100.6 cm</span>
+    </div>
+    <div>
+      <span>Diet</span>
+      <span>Omnivore</span>
+      <span>nuts, fruits, eggs</span>
+    </div>
+  </CardBody>
+  <CardFooter>
+    <Button>Share</Button>
+  </CardFooter>
+</Card>
+```
```

Comment:
What do we plan on doing if someone passes children other than `CardPreview`, `CardHeader`, `CardBody` or `CardFooter` to the `Card`?

Reply:
I think other children are allowed and expected, as CardPreview, CardHeader, CardBody or CardFooter for example might be wrapped by a data fetching component

Reply:
The idea is to have the inner components apply any self styling so it's up to the user to arrange the components as intended. This opens up room for the user to put a CardPreview on top with a CardHeader under it (or vice-versa) and for you to create your own special card components, giving you the freedom to customize.
Another option would be to have slots in the card for Header and Footer and have everything else just render in the middle, restricting usage.
Thoughts?

Reply:
I think freedom to customize makes sense, you can disregard my original comment.